src/wallets/hardware/LedgerWallet.ts

```diff
@@ -149,14 +153,13 @@ export default class LedgerWallet extends HardwareWallet implements ILedgerWalle
 
     this.transport.on("disconnect", (res: any) => {
       console.log(res);
-      EventHandler.callEventHandler("disconnect");
+      this.emitter.emit("disconnect", {});
```

Comment:
Could we remove the empty object here and for the others? We need to think about what the `data` values should be but for now, we can leave them blank (`undefined`).